.github/workflows/unit-tests.yml

```diff
@@ -78,6 +78,10 @@ jobs:
           echo PHPUNIT="$(pwd)/phpunit --exclude-group tty --exclude-group benchmark --exclude-group intl-data --exclude-group integration --exclude-group transient" >> $GITHUB_ENV
           echo COMPOSER_UP='composer update --no-progress --ansi'$([[ "${{ matrix.mode }}" != low-deps ]] && echo ' --ignore-platform-req=php+') >> $GITHUB_ENV
 
+          if [[ "${{ matrix.mode }}" = *-deps ]]; then
```

Comment:
see #61330, we need a PHPUnit version that supports the `--do-not-fail-on-deprecation` option